src/Jackett.Common/Definitions/exitorrent-org.yml

```diff
@@ -105,30 +131,25 @@ search:
       selector: a[href^="details.php?id="]
       attribute: href
     download:
-      selector: a[href^="download.php/"]
-      attribute: href
-    imdbid:
-      selector: a[href*="imdb.com/title/tt"]
+      selector: a[href^="download.php/"], a[href^="downloadfl.php/"]
       attribute: href
     genre:
-      selector: font[color="blue"]
+      selector: div.span_title span div
       filters:
-        - name: validate
-          args: "akcja, animacja, anime, biograficzny, czarna komedia, dokumentalny, dramat, dramat historyczny, edukacyjny, etiuda, familijny, fantasy, gangsterski, historyczny, horror, katastroficzny, komedia, komedia kryminalna, komedia obyczajowa, komedia romantyczna, krótkometrażowy, kryminał, melodramat, musical, muzyczny, niemy, przygodowy, romans, sci-fi, sensacyjny, sportowy, szpiegowski, thriller, western, wojenny"
+        - name: replace
+          args: ["Sci-Fi", "Sci_Fi"]
+        - name: replace
```

Comment:
We generally do not replace the dash. We replace spaces with underscore to prevent the two words being treated as separate keys.

Reply:
Ah damnit, I did this in TorrentMasters as well - e34716a8d9d32b57787bf15b1a36ef824733c037. I'm nearly sure I saw it being split as two different genres. brb...

Reply:
Nope, no idea why I thought that.